src/tezos_fmt_plug.c

```diff
@@ -141,6 +146,17 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 			sin[i] = (unsigned char*)salt[i];
 			pout[i] = seed[i].seed;
 			slens[i] = strlen(salt[i]);
+			static int warned;
```

Comment:
I'd personally put the declaration outside the parallel for (at, say, line 109) just to be sure.

Reply:
I meant line 113 (new code)

Reply:
I think there's no difference, but OK - I moved that line now.